symphony-bdk-config/src/main/java/com/symphony/bdk/core/config/model/BdkConfig.java

```diff
@@ -25,6 +25,7 @@ public class BdkConfig extends BdkServerConfig {
 
   private BdkRetryConfig retry = new BdkRetryConfig();
   private BdkDatafeedConfig datafeed = new BdkDatafeedConfig();
+  private BdkDatahoseConfig datahose = new BdkDatahoseConfig();
```

Comment:
Is a config with both DF and DH a valid config? Do we throw an exception if both are configured? Otherwise, which one should be used?

Reply:
Yes it is valid. Please mind that, when using the core, you have to manually start both loops and when using Spring Boot, only datafeed is enabled by default (you have to explicitly enable datahose). Documentation has been updated.